packages/azure-kusto-ingest/src/ingestClient.ts

```diff
@@ -50,7 +50,11 @@ export class KustoIngestClient extends KustoIngestClientBase {
         const blobName = generateBlobName(descriptor, props);
 
         const blockBlobClient = await this.resourceManager.getBlockBlobClient(blobName);
-        await blockBlobClient.uploadStream(descriptor.stream as Readable);
+        if (descriptor.stream instanceof Buffer) {
```

Comment:
This is actually not related to this PR but its a nice to have

Reply:
cool!